samples/ios/glean-sample-framework/sdk_generator.sh

```diff
@@ -0,0 +1,168 @@
+#!/bin/bash
```

Comment:
For the sample app we just used a symbolic link to the file.
We should do the same here (or maybe github is bad at showing that this is a symbolic link?)

Reply:
hm, no I just copied the file and didn't think about creating a symlink